### DISCUSSION TITLE
GitHub homepage no longer shows anything useful

### DISCUSSION THREAD

**Discussion by ghost** [community/community#66159] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Something is wrong with the homepage, it's no longer showing anything useful and is instead showing a bunch of random stuff that I'm not interested in

**Comment by gered** [comment]
Agreed. The previous home page feed showed items that were relevant to me. This one shows a bunch of random things that Github thinks I might be interested in. In practice, it's >50% random noise that pushes the stuff that I want to see out.

This change is a massive step backwards.

**Comment by RundesBalli** [comment]
I want the old home page feed back. With people I follow and nothing else.

**Comment by SpikeHD** [comment]
Just to give more specific points to why I and (maybe) others find the home page step back:

* My homepage is no longer showing me stars to my repositories. This may just be *past* stars, I don't think I've gotten any since the change 😛 so it's possible future stars show up fine, but it's a weird thing to be missing that was there before
* By default I was being shown, at the very top, a PR from a repo that I have never interacted with since starring many months ago. Just feels like it isn't very relevant, but I don't know how that's determined.
* Releases and announcements take up so much screen real estate. Maybe a more condensed version of these blocks could be nice?

I think overall, maybe just a bit more granularity in the filters would be a nice thing to see as well. I too prefer the old homepage right now, but I like to hope this one can be even better soon :)

**Comment by mryechkin** [comment]
Yup. All the useful info I _actually_ care about is gone. I don't need this to be a YouTube recommendations page, or some other shitty social network algorithm. I want to see stuff from people I **specifically** follow, in **chronological** order. That's it.

And let's not talk about how bad the new design is too. My eyes are trained to look at the left side of the screen for years now. Why am I now greeted with this giant piece of blank space - the exact spot where my eyes go first?? 🤦‍♂️ 

![GitHub Feed](https://github.com/community/community/assets/5657181/928e557c-5850-4241-a859-2e87209be187)

**Comment by henrybetts** [comment]
Unless I'm missing something, there's no option to see any activity on my (or my organisation's) repos like commits, pull requests etc.
I can only see posts from random repos that I am not particularly interested in.

This isn't great...

**Comment by ettaboyle** [comment]
_We've added the following update to the [Updates to your GitHub Feed](https://github.com/orgs/community/discussions/65343#top) Discussion Announcement originally posted August 29, 2003._


## _Sept. 12th Update_
_With the latest feed update on September 6, 2023, we made changes to the underlying technology of the feed to improve overall platform performance. As a result, we removed the functionality for “push events for repositories a user is subscribed to”. We don’t take these changes lightly, but as our community continues to grow tremendously, we have to prioritize our availability, user experience, and performance._ 


_Thanks to feedback from the community, we have updated our [changelog](https://github.blog/changelog/2023-09-06-updates-to-your-github-feed/) to clarify and have fixed the following bugs from the initial September 6th release:_


- _We were showing releases for organizations that you follow, instead of only showing them for repositories that you watch or star. This is now fixed._ 
- _We have also addressed a problem where 'Followed You' cards were sometimes appearing out of chronological order._
- _We do not plan to re-include “push events for repositories a user is subscribed to” for the reasons listed above, but we will continue to address feedback as it aligns with our platform goals._


_We understand that many of you are upset with the recent changes to your feed. We should have done a better job communicating recent changes and how those decisions relate to our broader platform goals. Your continued feedback is invaluable as we evolve and continue to strive to provide a first-class developer experience that helps every developer be happier and more productive._

**Comment by SpikeHD** [comment]
I was hoping for more coming from this update, honestly. I have been noticing more and more the shortcomings that other people have pointed out, and I still am not seeing stars for my own repositories, lol. Given how much it seems GitHub wants to turn the homepage into a social media feed, they're really missing out on the biggest dopamine hit they could provide 😛.

I still continue to believe that the new page does not properly portray information to the majority of users who simply want to see activity as it relates to their own work, or work they are involved in. I don't care the ReactOS had a new PR submitted yesterday, I have never contributed to that project, and now it's taking up half of my screen. I want to see what's going on with *my* work, or work that is *actually* important to me, in a more concise way like it was before. Not everything I star is an "I want to know every single change/release/etc. that happens in this repo", I thought that was what "Watching" was for.

One thing that I am perfectly fine with, though it should ***definitely be optional***, is showing recommended/trending repositories in the feed. I am all for increasing discovery of other peoples work, especially because those usually *do* tend to be things I would somewhat be interested in. I've mentioned this above already, but more granularity and options is something I feel should definitely be a thing.

**Comment by huaifeng1993** [comment]
The new homepage is just a shit for me!!! give me back the old one!
